modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -284,31 +201,86 @@ Runtime Fabric is not deployed behind a load balancer, these values should not b
+
+.. From Anypoint Platform select *Runtime Manager*.
+.. Select *Runtime Fabric*.
+.. Select the *Inbound Traffic* tab, and then select *Logs*.
+.. Select *Add Filter*.
+.. In the *IP* field, enter a single IP address or subset of addresses using CIDR notation.
+.. Select the log level to apply to this IP filter.
+.. Select *OK*.
+
+. Select *Deploy* to deploy the internal load balancer.
++
+The deployment can take up to a minute to complete. If successful, a message is displayed. You can view the deployment status at the beginning of the page. When the status transitions to *Applied*, the internal load balancer is successfully deployed and inbound traffic is enabled.
+
+== Step 3: Verify that Inbound Traffic is Enabled
+To resolve the CN of applications deployed on Runtime Fabric, DNS must be configured to map the CN to the IP address of the external load balancer or of each controller VM. 
+
+To test inbound traffic for applications before configuring DNS, send a request to the application using the IP address and a host header set to the domain.
+
+The structure of the domain depends on whether a wildcard was used in the CN of your certificate, for example:
+
+* A CN with a wildcard (e.g. *.example.com) uses the following request header format: `Host: {app-name}.example.com`. Use the following cURL command for verification:
++
+----
+curl -Lvk -XGET https://{ip-address}/{path} -H 'Host: {app-name}.example.com'
+----
+
+* A CN without a wildcard (e.g. example.com) uses the following request header format: `Host: example.com`. Use the following cURL command for vierfication:
++
+----
+curl -Lvk -XGET https://{ip-address}/{app-name}/{path} -H 'Host: example.com'`
+----
+
+== Step 4: Configure an External Load Balancer
+After you enable inbound traffic, you must configure Runtime Fabric to route incoming traffic to each enabled application in order for clients to send requests to deployed applications. 
+
+You must configure an external load balancer to load balance HTTPS traffic between each controller VM on Runtime Fabric. Controller VMs are virtual machines dedicated to run the components that power Anypoint Runtime Fabric. Each controller VM runs a replica of the internal load balancer and is configured to listen on port 443. 
+
+=== External Load Balancer Requirements
+When running multiple controller VMs, you must have an external load balancer outside Runtime Fabric to front each of the controller VMs.
+
+The external load balancer must support TCP load balancing and must be configured with a server pool containing the IP addresses of each controller VM. A health check must also be configured on the external load balancer, listening on port 443.
```

Comment:
```suggestion
The external load balancer must support TCP load balancing and must be configured with a server pool containing the IP addresses of each controller VM. A health check must also be configured on the external load balancer, listening on Port 443.
```

Reply:
Intentional passive voice? There is quite a bit of it in these topics. Just keep an eye on it!  :-)